pytripgui/plan_vc/plan_view.py

```diff
@@ -0,0 +1,90 @@
+from PyQt5 import QtWidgets
+from PyQt5.QtCore import Qt
+from PyQt5.QtWidgets import QTreeWidgetItem
+from pytripgui.view.gen.plan import Ui_PlanDialog
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+class PlanQtView(object):
+    """
+    """
+    def __init__(self):
+        self.ui = Ui_PlanDialog()
+        self.dialog = QtWidgets.QDialog()
+
+        self.ui.setupUi(self.dialog)
+        self._setup_callbacks()
+
+    def show(self):
+        self.dialog.show()
+        self.dialog.exec_()
+
+    def exit(self):
+        self.dialog.close()
+
+    def set_ok_callback(self, fun):
+        self.ui.accept_buttonBox.accepted.connect(fun)
+
+    def set_cancel_callback(self, fun):
+        self.ui.accept_buttonBox.rejected.connect(fun)
+
+    def _setup_callbacks(self):
+        pass
+
+    def get_basename_value(self):
+        return self.ui.basename_lineEdit.text()
+
+    def set_basename_value(self, basename):
+        self.ui.basename_lineEdit.setText(basename)
+
+    def get_comment_value(self):
+        return self.ui.comment_lineEdit.text()
+
+    def set_comment_value(self, comment):
```

Comment:
What about translating it to:
```
@comment_value.setter
def comment_value(self, comment):
   self.ui.comment_lineEdit.setText(comment)
```
?

Then an usage will be:
```
view.comment_value = model.comment
```
instead of
```
view.set_comment_value(model.comment)
```

Same can be done to setters.

Reply:
Just for historical context: previously both pytrip and pytripgui had lost of those Java "get_" and "set_" methods. I spend much time getting rid of these, see e.g. https://github.com/pytrip/pytrip/issues/118
I would be happy if we can avoid reintroducing them again. :)

Reply:
https://docs.quantifiedcode.com/python-anti-patterns/correctness/implementing_java-style_getters_and_setters.html

Reply:
done

Reply:
nice work @soundpl